views/login.handlebars

```diff
@@ -1,14 +1,15 @@
-<section id="create-login" class='text-center'>
-
-  <div class='buddy-card p-0 w-100'></div>
-  <div class='buddy-card p-0 w-100'></div>
-
+<div class='signup-images container text-center'>
+  <img src='nav-logo.png' />
+  <p id='signup-text'>where no buddy gets left behind...</p>
+</div>
+<div class='signup container text-center'>
   <form>
-    <div>
+    <header class='container text-center'></header>
+    <div id='create-account'>
```

Comment:
this is the same id as line 20/21 which you shouldn't do and its also creating a bug on this page where if you click anywhere in this div it fires off the create accounting function